From 80d02798fd2c006de271476f8413e2527ef93b72 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 28 Sep 2018 15:30:09 +0800
Subject: [PATCH 4757/5242] MLK-15975-2: ARM: dts: change dual fifo script to
 multi fifo script

commit  0faeea35203038b3c1741452fe7ffd4e9df397d7 from
https://source.codeaurora.org/external/imx/linux-imx.git

change dual fifo script to multi fifo script for SSI.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit 417c76c51f0fde7b66cd499e405b1465df18da57)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 62abf23..10b5dcf 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -456,7 +456,7 @@
 					clocks = <&clks IMX6SX_CLK_SSI1_IPG>,
 						 <&clks IMX6SX_CLK_SSI1>;
 					clock-names = "ipg", "baud";
-					dmas = <&sdma 37 22 0>, <&sdma 38 22 0>;
+					dmas = <&sdma 37 26 0>, <&sdma 38 26 0>;
 					dma-names = "rx", "tx";
 					fsl,fifo-depth = <15>;
 					status = "disabled";
@@ -470,7 +470,7 @@
 					clocks = <&clks IMX6SX_CLK_SSI2_IPG>,
 						 <&clks IMX6SX_CLK_SSI2>;
 					clock-names = "ipg", "baud";
-					dmas = <&sdma 41 22 0>, <&sdma 42 22 0>;
+					dmas = <&sdma 41 26 0>, <&sdma 42 26 0>;
 					dma-names = "rx", "tx";
 					fsl,fifo-depth = <15>;
 					status = "disabled";
@@ -484,7 +484,7 @@
 					clocks = <&clks IMX6SX_CLK_SSI3_IPG>,
 						 <&clks IMX6SX_CLK_SSI3>;
 					clock-names = "ipg", "baud";
-					dmas = <&sdma 45 22 0>, <&sdma 46 22 0>;
+					dmas = <&sdma 45 26 0>, <&sdma 46 26 0>;
 					dma-names = "rx", "tx";
 					fsl,fifo-depth = <15>;
 					status = "disabled";
-- 
1.7.9.5

